assets/readme.md

```diff
@@ -1,19 +1,121 @@
+
+Reveal provides you a company profile by performing a reverse IP lookup which provides your sales team the account information, but you can go one step further by automating the process to find prospects for this account. More details, see "How does the Trigger for Prospection work?".
+
+**How to enable Anonymous Users for your organization?** 
+You can enable anonymous users or check the feature status within your **Organization Settings**. To access your organization settings, click on Settings in the main menu bar and select the (User) Settings from the menu to the left:
+![Reveal 2](./docs/reveal02.png)
+
+**How does the Trigger for Prospection work?**
+The trigger let’s you define when the Clearbit Connector automatically executes the Prospector workflow to find contacts for a revealed account.
+You can trigger prospection as soon as the account is revealed by setting the slider to one, or you can require a certain amount of different users visiting your website from the same IP address by adjusting the slider. Selecting a higher value will help to preserve credits but you might not get directly prospects for an account.
+Please note, this will only have an effect if the [Prospector](#Prospector) feature is enabled.
+
+## Prospector
+
+The Prospector feature of the connector enables you to grow your pipeline and find the right buyer within a named account by passing along the domain name.
+
+You can enable or disable the feature by toggling the checkbox in the **Prospector** section of the Settings tab:
+![Prospector 1](./docs/prospector01.png)
+
+The Prospector will search for contacts within named accounts by passing the domain name of the website along. You can define which attribute you want to use as company domain by picking it from the dropdown list.
+If you want to target only accounts that match a certain profile, you can create segments for targeted profiles and restrict the Prospector flow to run only for accounts within those segments.
+You can define the maximum number of leads to fetch from Clearbit by adjusting the slider to desired position. This setting will help you to control your costs.
+
+While the general settings control the overall behavior of the connector, you can refine your search by filtering a certain role, seniority, title or any combination of the aforementioned. This will help you to find the people matching your ICP within an account:
+![Prospector 2](./docs/prospector02.png)
+
+**When does the Prospector workflow run?**
+The Prospector workflow gets triggered exactly once per user, if ***all*** of the following conditions are met:
+
+- The user/account does not have the attribute `clearbit/prospected_at` set.
+- There are no named users in your Hull organization with the same domain.
+- The number of anonymous users has reached the threshold defined via the trigger in the [Reveal](#Reveal) configuration.
```

Comment:
`The number of anonymous users` -> `The number of anonymous users with the same company domain`